src/main/java/io/github/thebusybiscuit/slimefun4/implementation/operations/CraftingOperation.java

```diff
@@ -40,7 +40,15 @@ public CraftingOperation(@Nonnull ItemStack[] ingredients, @Nonnull ItemStack[]
     @Override
     public void addProgress(int num) {
         Validate.isTrue(num > 0, "Progress must be positive.");
-        currentTicks += num;
+
+        /**
+         * Normalize the tickrate to the config value custom-ticker-delay.
+         * This makes the machines run nearly the same speed on changing the delay.
+         */
```

Comment:
Is there a reason this is a block comment rather than just normal ones? Not a problem just a curiosity

Reply:
Honestly no clue I didn’t write it